optimum/intel/openvino/modeling.py

```diff
@@ -369,6 +369,13 @@ class OVModelForFeatureExtraction(OVModel):
     auto_model_class = AutoModel
 
     def __init__(self, model=None, config=None, **kwargs):
+        if {"token_embeddings", "sentence_embedding"}.issubset(
+            {name for output in model.outputs for name in output.names}
+        ):  # Sentence Transormers outputs
+            raise ValueError(
+                "This model is Sentence Tranfromers converted model. Please use OVSentenceTransformer explicitly for this model."
```

Comment:
```suggestion
                "This model is a Sentence Transformers model. Please use `OVSentenceTransformer` to load this model."
```

Reply:
It requires editing the test `test_modeling.py::OVModelForFeatureExtractionIntegrationTest`. Fixed

Reply:
thanks for the fix !